plugins/modules/irc.py

```diff
@@ -195,7 +195,8 @@ def send_msg(msg, server='localhost', port='6667', channel=None, nick_to=None, k
 
     irc = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
     if use_ssl:
-        irc = ssl.wrap_socket(irc)
+        context = ssl.create_default_context()
+        irc = context.wrap_socket(irc)
```

Comment:
This is likely a backwards-compatible breaking change, since the context internally created by `ssl.wrap_socket` was rather insecure (see https://docs.python.org/3.11/library/ssl.html#ssl.wrap_socket). The context created by `ssl.create_default_socket` (https://docs.python.org/3/library/ssl.html#ssl.create_default_context) is secure by default.

Reply:
Fixed by 054d21d986c891fdaf076768d61750ed7c4cf72f.